internal/controller/k0smotron.io/k0smotroncluster_statefulset.go

```diff
@@ -118,7 +118,7 @@ func (r *ClusterReconciler) generateStatefulSet(kmc *km.Cluster) (apps.StatefulS
 						Name:            "controller",
 						Image:           kmc.Spec.GetImage(),
 						ImagePullPolicy: v1.PullIfNotPresent,
-						Args:            []string{"/k0smotron-entrypoint.sh"},
+						Args:            []string{"/bin/sh", "-c", "/k0smotron-entrypoint.sh"},
```

Comment:
Would have `[]string{"/bin/sh", "/k0smotron-entrypoint.sh"}` (without the -c) fixed the problem as well? I really don't have a clue what the root cause may be :thinking: